Sources/Internal.swift

```diff
@@ -558,13 +558,14 @@ extension String {
     /// // prints "50334ee0b51600df6397ce93ceed4728c37fee4e"
     /// ```
     var sha1: String? {
-        guard let input = self.data(using: .utf8) else {
-            return nil
-        }
-        var hash = [UInt8](repeating: 0, count: Int(CC_SHA1_DIGEST_LENGTH))
-        input.withUnsafeBytes {
-            _ = CC_SHA1($0, CC_LONG(input.count), &hash)
+        guard let input = self.data(using: .utf8) else { return nil }
+        
+        let hash = input.withUnsafeBytes { (bytes: UnsafeRawBufferPointer) -> [UInt8] in
```

Comment:
@kean would you prefer an `#if compiler(>=5.0)` here?

Reply:
Please, that would be ideal. I intend to keep Swift 4.2 support for the next year or so.

Reply:
Updated. Added `#if swift(>=5.0)` check.